devel/icu/Portfile

```diff
@@ -106,17 +125,19 @@ subport ${name}-lx {
 }
 
 if {${subport} eq ${name}} {
-    revision                2
+    revision                0
```

Comment:
The other subports should set an explicit revision too, so that revision changes to icu don't inadvertently change the revisions of the others.

Reply:
Thank you for noticing.
This should now be fixed.